src/ResourceManager/Insights/Commands.Insights/Autoscale/NewAzureRmAutoscaleRuleCommand.cs

```diff
@@ -29,15 +29,6 @@ public class NewAzureRmAutoscaleRuleCommand : AzureRMCmdlet
         private readonly TimeSpan MinimumTimeWindow = TimeSpan.FromMinutes(5);
         private readonly TimeSpan MinimumTimeGrain = TimeSpan.FromMinutes(1);
 
-        /// <summary>
-        /// Scale type enum.
-        /// Keep this enum for backwards compatibility. The only value supported is ChangeCount
-        /// </summary>
-        public enum ScaleType
```

Comment:
The value now comes from the SDK, it supports these values: ChangeCount (default), PercentChangeCount, ExactCount.
Line 99 of the file documents this, and line 117 warns the user that there are two more accepted values now.